package.json

```diff
@@ -741,7 +741,13 @@
                 "title": "Get Release",
                 "description": "Get a Helm release from the cluster",
                 "category": "Helm"
-            }
+            },
+            {
+				"command": "extension.helmPackage",
```

Comment:
I wonder if we should also add this to the file explorer context menu (a la Inspect Values) but I'm not sure whether there is a good criterion for which nodes to show it on -- happy to defer.

Reply:
Can you explain more please ?